examples/src/main/java/io/numaproj/numaflow/examples/function/map/evenodd/EvenOddFunction.java

```diff
@@ -1,10 +1,10 @@
 package io.numaproj.numaflow.examples.function.map.evenodd;
 
-import io.numaproj.numaflow.function.Datum;
 import io.numaproj.numaflow.function.FunctionServer;
-import io.numaproj.numaflow.function.Message;
-import io.numaproj.numaflow.function.MessageList;
-import io.numaproj.numaflow.function.map.MapHandler;
+import io.numaproj.numaflow.function.handlers.MapHandler;
```

Comment:
Looks like you guys need to use a unified IDE :)

Reply:
no this was because I refactored the packages.